themis.py

```diff
@@ -34,6 +34,15 @@ def hello_command(update: Update, context: CallbackContext) -> None:
     print(f'User {update.effective_user.first_name} called /hello')
     update.message.reply_text(f'Hello {update.effective_user.first_name}!')
 
+"""
+Command /time =)
+"""
+def time_command(update: Update, context: CallbackContext) -> None:
+    print(f'User {update.effective_user.first_name} called /time')
+    the_datetime = datetime.now()
+    currnet_time = the_datetime.time()
+    update.message.reply_text(f'The time is {currnet_time}')
```

Comment:
To avoid printing milliseconds you can use strftime function to format time as you want.
For example:
update.message.reply_text(f'Current time is {currnet_time.strftime("%H:%M:%S")}')